### PR TITLE
DBZ-6712 fix snapshot to history internal topic to respect schema.history.internal.store.only.captured.databases.ddl flag

### PR DESCRIPTION
**Pull request by matan-cohen**
more details in [DBZ-6712](https://issues.redhat.com/browse/DBZ-6712)

### CODE REVIEW

**Comment by jpechane** [comment]
@matan-cohen Thanks for the PR! Would it be please possible to add tests that verify the expected behaviour?

**Comment by matan-cohen** [comment]
@jpechane  Added tests.

File: jenkins-jobs/scripts/config/Aliases.txt [hunk 1]
```diff
@@ -227,3 +227,4 @@ tyrantlucifer,Chao Tian
 ryanvanhuuksloot,Ryan van Huuksloot
 vsantona, Vincenzo Santonastaso
 rolevinks, Stein Rolevink
+matan-cohen,Matan Cohen
```
Comment (rk3rn3r) [inline jenkins-jobs/scripts/config/Aliases.txt:230]:
```suggestion
matan-cohen, Matan Cohen
```

**Comment by jpechane** [comment]
@matan-cohen Thanks. I create a new PR and squased your commits into a single one - https://github.com/debezium/debezium/pull/4788

I'll merge it whe CI passes

**Pull request closed without merging**